app/instance-initializers/ember-intl.js

```diff
@@ -10,6 +10,7 @@ import { filterBy } from 'ember-intl/utils/initialize';
 export function instanceInitializer(instance) {
     let service = instance.container.lookup('service:intl');
 
+    service.set('appInstance', instance);
```

Comment:
#127 it all seems to be working except for tests and one other comment I pointed out which we need to guard against for backwards compat